### ISSUE TITLE
phpCAS warning

### ISSUE THREAD

**Issue by vbanos** [AlexandriaUP/Alexandria#1]
Σας προτείνω να μην περιλαμβάνετε το phpCAS στον κώδικα της εφαρμογής σας
αλλά να το εγκαθιστά ο χρήστης εκτελώντας το composer κατά την εγκατάσταση.
Γενικά αυτή η λύση έχει πολλά πλεονεκτήματα.

Επιπλέον, έτσι όπως το έχετε τώρα εμφανίζεται το εξής warning:
```
[Mon Jan 09 13:18:07.615418 2023] [proxy_fcgi:error] [pid 2134785:tid 140431429969664] [client 94.68.50.131:53660] AH01071: Got error 'PHP message: PHP Deprecated:  phpCAS autoloader is deprecated. Install phpCAS using composer instead. in /home/alexandria/public_html/uppa_core/cas/phpCAS/CAS/Autoload.php on line 86', referer: https://alexandria.ihu.edu.gr/
```

**Comment by vbanos** [comment]
Ξέχασα να γράψω ότι το τρέχω σε PHP 8.1.13 και ubuntu 20.04 LTS.

**Comment by AlexandriaUP** [comment]
Ναι, είναι στον προγραμματισμό μας να ετοιμάσουμε μια νέα έκδοση όπου όλες οι third-party βιβλιοθήκες θα εγκαθίστανται με τον composer.

**Comment by AlexandriaUP** [comment]
Στο main branch, η διαχείριση 3rd party βιβλιοθηκών γίνεται πλέον μέσω composer. Οπότε, αν θέλει κανείς, μπορεί να κάνει require την phpCAS και να προστεθεί στο composer.json
